http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20240605_add_update_app_log_table.xml

```diff
@@ -0,0 +1,38 @@
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="jcanas" id="add_update_app_log_table">
+
+        <createTable tableName="UPDATE_APP_LOG">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="jobid" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="appId" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="appErrorId" type="BIGINT">
+                <constraints nullable="true"/>
+            </column>
+            <column name="status" type="ENUM('ERROR', 'RUNNING', 'SUCCESS')" defaultValue="RUNNING">
+                <constraints nullable="false"/>
+            </column>
+            <column name="startTime" type="TIMESTAMP(6)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="endTime" type="TIMESTAMP(6)">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+        <createIndex indexName="FK_UPDATE_APP_LOG_APP_ID" tableName="UPDATE_APP_LOG">
+            <column name="appId"/>
+        </createIndex>
+        <createIndex indexName="FK_UPDATE_APP_LOG_ID" tableName="UPDATE_APP_LOG">
+            <column name="id"/>
+        </createIndex>
+        <addUniqueConstraint columnNames="jobId, appId" constraintName="IDX_APP_ID_PER_JOB_UNIQUE" tableName="UPDATE_APP_LOG"/>
+        <addUniqueConstraint columnNames="appErrorId" constraintName="IDX_APP_ERROR_UNIQUE" tableName="UPDATE_APP_LOG"/>
```

Comment:
if appErrorId were a foreign key I think this could be removed

Reply:
looks like adding this explicitly isn't wrong tho, foreign keys apparently don't have to be unique. but mb since it's based on a unique id in the other table it's not needed here